project/PY-Chang/README.md

```diff
@@ -0,0 +1,59 @@
+# YAMM
+Yet Another Matrix Multiplier!
+
+## Basic Information
+> Github repository: https://github.com/PY-Chang/YAMM.git
+
+This is a matrix multiplier based on C++ and Python hybrid system. This project implements the classic Strassen algorithm with the help of parallel computing to achieve better performance on modern computer.
+
+## Problem to Solve and Prospective Users
+Matrix multiplication is one of the most important linear algebra operation in the modern computing world. For decades, many researches have been invested to improve the efficiency of matrix multiplication. 
+
+In 1969, Volker Strassen developed an algorithm, namely, Strassen algorithm which redeuces the time complicity of the matrix multiplication down to around O(n<sup>2.8074</sup>) in the first time. 
+
+
+Although in recent years researchers have developed faster algorithms than Strassen's, those algorithms have been viewed as galactic algorithms which means non-practical to present-day computers.
```

Comment:
This is not true.  Advanced algorithms are used in commercial tools already.  They are, oftentimes, not worth the time for being implemented in code for researches in fields other than linear algebra itself.